deployment/helm/node-feature-discovery/templates/clusterrolebinding.yaml

```
@@ -32,3 +32,21 @@ subjects:
   name: {{ include "node-feature-discovery.topologyUpdater.serviceAccountName" . }}
   namespace: {{ include "node-feature-discovery.namespace" .  }}
 {{- end }}
+
+---
+{{- if and (eq .Values.topologyGC.enable true) (eq .Values.topologyGC.rbac.create true) ( eq .Values.topologyUpdater.enable true ) }}
```

nit: could be simplified

```suggestion
{{- if and .Values.topologyGC.enable .Values.topologyGC.rbac.create .Values.topologyUpdater.enable }}
```

---

done